bandit/core/context.py

```diff
@@ -199,50 +200,56 @@ def _get_literal_value(self, literal):
         :param literal: The AST literal to convert
         :return: The value of the AST literal
         '''
-        if isinstance(literal, _ast.Num):
+        if isinstance(literal, ast.Num):
```

Comment:
On Python 3.8
> AttributeError: module 'ast' has no attribute 'Num'

Reply:
`ast.Num` exists on Python 3.8. `_ast.Num` is removed.